hack/central-repo/upload-plugins.sh

```diff
@@ -125,7 +125,8 @@ addPlugin() {
 
 k8sPlugins=(cluster feature management-cluster package secret telemetry kubernetes-release)
 tmcPlugins=(account apply audit cluster clustergroup data-protection ekscluster events iam 
-            inspection integration management-cluster policy workspace)
+            inspection integration management-cluster policy workspace helm secret 
+            continuousdelivery tanzupackage)
```

Comment:
These are new plugins added by the TMC endpoint.  I'm adding them to the docker test central repo so that a plugin sync can find all plugins and succeed.